scenes/scene.yaml

```diff
@@ -1,19 +1,19 @@
 global:
-    default_order: function() { return feature.sort_key || 0; }
+    feature_order: function () { return feature.sort_rank; }
 
 fonts:
     Montserrat:
         url: https://fonts.gstatic.com/s/montserrat/v7/zhcz-_WihjSQC0oHJ9TCYL3hpw3pgy2gAi-Ip7WPMi0.woff
     Open Sans:
-        - weight: 300
-          url: fonts/open sans-300normal.ttf
-        - weight: 300
+        - weight: 400
+          url: http://fonts.gstatic.com/s/opensans/v13/IgZJs4-7SA1XX_edsoXWog.ttf
```

Comment:
Those are quite simple fonts with only a few hundreds kilobytes in it (at most 200k), there's not much gain we could have. Finding other formats that `woff2` is quite tedious since they don't propose it anymore on their website https://fonts.google.com/. I had to use an API key and download a full list of their fonts with TTF formats from https://www.googleapis.com/webfonts/v1/webfonts:

[GoogleFonts.txt](https://github.com/tangrams/tangram-es/files/694178/GoogleFonts.txt)